### PR TITLE
homeWork2

### PR DESCRIPTION
**Pull request by ViacheslavGIT**

### CODE REVIEW

**Comment by roskyba** [comment]
Good work! Approved MR!

**Review by roskyba — APPROVED** [review]

File: 2/index.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+let myСity = "Dnipro";
+console.log(myСity);
+
+let average = (1 + 2 + 3 + 4) / 4;
+console.log(average);
+
+let radiusСircle = 4;
+console.log(radiusСircle);
+
+let perimeterСircle = 2 * radiusСircle * 3.14;
```
Comment (roskyba) [inline 2/index.js:16]: Better to use `Math.PI` number, since it is more accurate

File: 2/index.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+console.log(perimeterСircle);
+
+let areaСircle = 3.14 * Math.pow(radiusСircle, 2); //remembered manematics
```
Comment (roskyba) [inline 2/index.js:19]: Good job while using `Math.pow()`

**Review by roskyba — APPROVED** [review]

File: 2/index.js (hunk 1 above)
Comment (ViacheslavGIT) [reply 2/index.js:16]: ok

**Pull request merged**